scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler

```diff
@@ -62,14 +71,28 @@ fi
 bundle_id=${1:-$HOSTNAME}
 dest_dir=${2:-$DEFAULT_DEST_DIR}
 
-while getopts ":b:t:c:s:" flag
-do
-    case "${flag}" in
-          b) bundle_id="$OPTARG";;
-          t) dest_dir="$OPTARG";;
-          c) config_url="$OPTARG";;
-          s) sevice="$OPTARG";;
-    esac
+duration=P5d
```

Comment:
These are variables need to mention as "longoptions"  in getopts
kindly change this script similar to this, using longoptions and options.

TEMP=$( getopt --options h:c \
                   --longoptions help,verbose,duration:,size_limit:,services:,binlogs:,coredumps:,stacktrace: \
                   --name "$PROG_NAME" -- "$@")
[[ $? -ne 0 ]] && usage

eval set -- "$TEMP"

while true ; do
        case "$1" in
            --duration)        duration_cal=$(echo $2 | sed -e 's/,/\n/g') ; shift 2 ;;
            --size_limit)       size_cal=$(echo $2 | sed -e 's/,/\n/g') ; shift 2 ;;
            --services)         service_list=$2; shift 2 ;;
                                set IFS='|'
                                for service in $service_list 
                                 do
                                     /* read each services name and set a flag true for that service */
                                 done
                /*** add other bundle options ***/                 
                esac
done